performance/performance_tuning.rst

```diff
@@ -35,7 +34,62 @@ PostgreSQL tuning
 
 The Citus master partitions an incoming query into fragment queries, and sends them to the workers for parallel processing. The workers are just extended PostgreSQL servers and they apply PostgreSQL's standard planning and execution logic for these queries. So, the first step in tuning Citus is tuning the PostgreSQL configuration parameters on the workers for high performance.
 
-This step involves loading a small portion of the data into regular PostgreSQL tables on the workers. These tables would be representative of shards on which your queries would run once the full data has been distributed. Then, you can run the fragment queries on the individual shards and use standard PostgreSQL tuning to optimize query performance.
+While adjusting parameters to tune the workers we want fast iterations. To begin the tuning process, create a Citus cluster and load a small portion of your data in it. This will keep operations fast. Once the data is loaded, use the EXPLAIN command on the master node to inspect performance. 
+
+Citus extends the SQL EXPLAIN command to provide information about simple distributed queries, meaning queries that do not contain re-partition jobs. The EXPLAIN output shows how each worker is processing the query and also a little about how the master node is combining all the results.
+
+Here is an example of explaining the plan for a particular query in the :ref:`hash partitioning tutorial <tut_hash>`:
+
+::
+
+  explain SELECT comment FROM wikipedia_changes c, wikipedia_editors e
+          WHERE c.editor = e.editor AND e.bot IS true LIMIT 10;
+
+  Distributed Query into pg_merge_job_0005
+    Executor: Real-Time
+    Task Count: 16
+    Tasks Shown: One of 16
+    ->  Task
+      Node: host=localhost port=9701 dbname=j
+      ->  Limit  (cost=0.15..6.87 rows=10 width=32)
+        ->  Nested Loop  (cost=0.15..131.12 rows=195 width=32)
+          ->  Seq Scan on wikipedia_changes_102024 c  (cost=0.00..13.90 rows=390 width=64)
+          ->  Index Scan using wikipedia_editors_editor_key_102008 on wikipedia_editors_102008 e  (cost=0.15..0.29 rows=1 width=32)
+            Index Cond: (editor = c.editor)
+            Filter: (bot IS TRUE)
+  Master Query
+    ->  Limit  (cost=0.00..0.00 rows=0 width=0)
+      ->  Seq Scan on pg_merge_job_0005  (cost=0.00..0.00 rows=0 width=0)
+  (15 rows)
+
+This tells you several things. To begin with there are sixteen shards, and we're using the real-time Citus executor setting:
+
+::
+
+  Distributed Query into pg_merge_job_0005
+    Executor: Real-Time
+    Task Count: 16
+
+Next it picks one of the workers to and shows you more about how the query behaves there. It indicates the host, port, and database so you can connect to the worker directly if desired:
+
+::
+
+    Tasks Shown: One of 16
+    ->  Task
+      Node: host=localhost port=9701 dbname=j
```

Comment:
Can we use a different dbname other than "j"? Maybe something like "postgres" as a default?